test/helpers/kubectl.go

```diff
@@ -1648,11 +1648,18 @@ func (kub *Kubectl) GatherLogs() {
 		"kubectl describe pods --all-namespaces":                     "pods_status.txt",
 		"kubectl get replicationcontroller --all-namespaces -o json": "replicationcontroller.txt",
 		"kubectl get deployment --all-namespaces -o json":            "deployment.txt",
-		"kubectl get all,cnp,cep --all-namespaces -o wide":           "kubernetes_summary.txt",
 	}
 
 	kub.GeneratePodLogGatheringCommands(reportCmds)
 
+	res := kub.Exec(fmt.Sprintf(`%s api-resources | grep -v "^NAME" | awk '{print $1}'`, KubectlCmd))
+	if res.WasSuccessful() {
+		for _, line := range res.ByLines() {
+			key := fmt.Sprintf("%s get %s --all-namespaces -o wide", KubectlCmd, line)
+			reportCmds[key] = fmt.Sprintf("api-resource-%s.txt", line)
+		}
+	}
+
```

Comment:
Log an error to console / log file to indicate what the output of `res` was if it was not successful.

Reply:
Minor typo - api-resoureces. Not blocking merging though, can be fixed later.